omniledger/service/service.go

```diff
@@ -250,6 +251,7 @@ func (s *Service) SetPropagationTimeout(p time.Duration) {
 	s.storage.PropTimeout = p
 	s.storage.Unlock()
 	s.save()
+	s.skService().SetBFTTimeout(p)
```

Comment:
I'm not sure whether this should be here. The BFT timeout is different from the propagation timeout. The BFT timeout is like a time budget, where every sub-protocol in ByzCoinX will use some of the budget. The propagation timeout is always from the `message.Propagate` protocol. I think these two should be set independently.